standard/template/sections/clause_9_zarr_encoding.adoc

```diff
@@ -0,0 +1,16 @@
+== Unified Data Model Encoding for Zarr
+
+This clause defines the encoding of the unified data model into the Zarr format. The encoding supports both Zarr Version 2 and Zarr Version 3.
```

Comment:
This is a good document. I suggest we compartmentalize all of the details of Zarr encoding and the differences between V2 and V3 in this page, and elsewhere we simply refer to the logical Zarr entities (Array, Group, Attributes, etc.)

We should also reference the Zarr spec pages directly here and explain that the GeoZarr spec is not the canonical reference for Zarr encoding.